SpatialGDK/Source/SpatialGDK/Private/Utils/CheckoutRadiusConstraintUtils.cpp

```diff
@@ -0,0 +1,169 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "Utils/CheckoutRadiusConstraintUtils.h"
+
+#include "UObject/UObjectIterator.h"
+
+#include "SpatialGDKSettings.h"
+
+DEFINE_LOG_CATEGORY(LogCheckoutRadiusConstraintUtils);
+
+namespace SpatialGDK
+{
+
+QueryConstraint CheckoutRadiusConstraintUtils::GetDefaultCheckoutRadiusConstraint()
+{
+	// Use AActor's ClientInterestDistance for the default radius (all actors in that radius will be checked out)
+	const AActor* DefaultActor = Cast<AActor>(AActor::StaticClass()->GetDefaultObject());
+
+	const float DefaultDistanceSquared = DefaultActor->NetCullDistanceSquared;
+	const float DefaultCheckoutRadius = FMath::Sqrt(DefaultDistanceSquared / (100.0f * 100.0f));
+
+	QueryConstraint DefaultCheckoutRadiusConstraint;
+	DefaultCheckoutRadiusConstraint.RelativeCylinderConstraint = RelativeCylinderConstraint{ DefaultCheckoutRadius };
+
+	return DefaultCheckoutRadiusConstraint;
+}
+
+TMap<UClass*, float> CheckoutRadiusConstraintUtils::GetActorTypeToRadius()
```

Comment:
Can we add a `using` for this container definition?

Reply:
Oh, it's different to the one used in `DedupeDistancesAcrossActorTypes`. Nevermind.